### ISSUE TITLE
Test this on faculty WordPress network site for use there

### ISSUE THREAD

**Issue by knice** [ucsc/ucsc-custom-functionality#13]

**Comment by Herm71** [comment]
This is being worked on. Issue #21 describes the work needed to do before we can test on faculty WP network.

**Comment by Herm71** [comment]
The work in Issue #21 has been merged into the main branch. I is ready for zipping up and testing on the faculty WordPress network.
